### PR TITLE
Revert making post title translatable.

### PR DESCRIPTION
**Pull request by kebbet**
Trac ticket: https://core.trac.wordpress.org/ticket/55392

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Comment by mukeshpanchal27** [comment]
Thanks for flagging the issue.

In core we use similar core reference tickets. There is no specific format for this type of document.

https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api/endpoints/class-wp-rest-menu-items-controller.php#L331
https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api/endpoints/class-wp-rest-menu-items-controller.php#L333
https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/blocks/gallery.php#L84

Let's update the message and use period at end.

`// Do not make string translatable, see https://core.trac.wordpress.org/ticket/54518.`

File: src/wp-includes/class-wp-theme-json-resolver.php [hunk 1]
```diff
@@ -342,7 +342,7 @@ public static function get_user_data_from_wp_global_styles( $theme, $create_post
 				array(
 					'post_content' => '{"version": ' . WP_Theme_JSON::LATEST_SCHEMA . ', "isGlobalStylesUserThemeJSON": true }',
 					'post_status'  => 'publish',
-					'post_title'   => __( 'Custom Styles' ),
+					'post_title'   => 'Custom Styles', // Do not make string translatable, see ticket https://core.trac.wordpress.org/ticket/54518.
```
Comment (mukeshpanchal27) [inline src/wp-includes/class-wp-theme-json-resolver.php]:
```suggestion
					'post_title'   => 'Custom Styles', // Do not make string translatable, see https://core.trac.wordpress.org/ticket/54518.
```

**Review by mukeshpanchal27 — COMMENTED** [review]
Remove the ticket as it was not used in other similar document references.

File: src/wp-includes/class-wp-theme-json-resolver.php (hunk 1 above)
Comment (kebbet) [reply src/wp-includes/class-wp-theme-json-resolver.php:345]: Thanks for noting!

**Comment by kebbet** [comment]
> Thanks for flagging the issue.
> 
> In core we use similar core reference tickets. There is no specific format for this type of document.
> 
> https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api/endpoints/class-wp-rest-menu-items-controller.php#L331 https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api/endpoints/class-wp-rest-menu-items-controller.php#L333 https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/blocks/gallery.php#L84
> 
> Let's update the message and use period at end.
> 
> `// Do not make string translatable, see https://core.trac.wordpress.org/ticket/54518.`

Thanks for your review @mukeshpanchal27. The two commits adds a preiod and removes `ticket` from the comment.

**Comment by kebbet** [comment]
Merged in https://core.trac.wordpress.org/changeset/54246

**Pull request closed without merging**